src/channel.c

```diff
@@ -99,12 +101,12 @@ IMPLEMENT_MOCKABLE_FUNCTION(, CHANNEL_RESULT, channel_pull, THANDLE(CHANNEL), ch
         CHANNEL* channel_ptr = THANDLE_GET_T(CHANNEL)(channel);
 
         /*Codes_SRS_CHANNEL_43_011: [ channel_pull shall call channel_internal_pull and return as it returns. ]*/
-        result = channel_internal_pull(channel_ptr->channel_internal, pull_callback, pull_context, out_op_pull);
+        result = channel_internal_pull(channel_ptr->channel_internal, correlation_id, pull_callback, pull_context, out_op_pull);
```

Comment:
>channel_internal_pull [](http://example.com/codeflow?start=17&length=21)

would be nice to have a log line stating that `channel_internal_pull` failed and with what error code.

Reply:
This is a pass through module. Error is printed in channel_internal.c